CRM/MembershipExtras/Upgrader.php

```diff
@@ -479,4 +479,55 @@ public function upgrade_0005() {
     return TRUE;
   }
 
+  public function upgrade_0006() {
+    $this->createIsActivePaymentPlanCustomGroupAndFields();
+
+    return TRUE;
+  }
+
+  /**
+   * Creates "Is Payment Plan active?"
+   * custom group and its "Is Active?"
+   * field.
+   *
+   * @throws CiviCRM_API3_Exception
+   */
+  private function createIsActivePaymentPlanCustomGroupAndFields() {
+    $customGroup = civicrm_api3('CustomGroup', 'get', [
+      'extends' => 'ContributionRecur',
+      'name' => 'payment_plan_is_active',
+    ]);
+
+    if (!$customGroup['count']) {
+      $customGroup = civicrm_api3('CustomGroup', 'create', [
+        'extends' => 'ContributionRecur',
+        'name' => 'payment_plan_is_active',
+        'title' => E::ts('Is Payment Plan active?'),
```

Comment:
Not sure if it is a good custom group name/title.
Maybe "Payment Plan Details/Attributes/Information" is better.

Reply:
yeah good point, In case we might need to add some other helpful flags in the future we need a better name, I think I will go with "Payment Plan extra attributes"